data.sql

```diff
@@ -9,4 +9,19 @@ INSERT INTO animals (
 VALUES ('Agumon', '03-02-2020', 0, true, 10.23),
   ('Gabumon', '15-11-2018', 2, true, 8),
   ('Pikachu', '07-01-2021', 1, false, 15.04),
-  ('Devimon', '12-05-2017', 5, true, 11);
```

Comment:
- [ ] When I run this file, I get this error:

![image](https://github.com/Abdulwhbhussain/Vet-clinic/assets/77106962/87e393c5-32d5-449f-9285-28db26a7b7d9)

You are inserting the date of birth for the animals in the day/month/year format. I would advise you to change the format to year/month/day format. Kindly implement this change. 👍 😎

Reply:
implemented